nilearn/decoding/decoder.py

```diff
@@ -175,9 +192,12 @@ def _parallel_fit(estimator, X, y, train, test, param_grid, is_classification,
             best_intercept = estimator.intercept_
             best_param = param
 
-    if (selector is not None) and do_screening:
+    if do_screening:
         best_coef = selector.inverse_transform(best_coef)
```

Comment:
This is the part where I changed in PR #2362 (related to issue #2360). I guess maybe we could change this directly in this PR?

Reply:
You mean adding : 
``
if len(best_coef.shape) == 1:
    best_coef = best_coef.reshape(1, -1)
``
to fix #2360?

Reply:
Yes adding this line indeed

Reply:
Given the discussions happening on #2362  I guess we can run both in parallel wdyt?

Reply:
thought about it again as those discussions are not related to this bugfixing. I'm thinking of including it.